lib/hedge_fund_interview/prisms/answer_interview_question.ex

```diff
@@ -32,10 +33,22 @@ defmodule HedgeFundInterview.Prisms.AnswerInterviewQuestion do
       {:error, error}
   end
 
-  defp build_prompt(input) do
+  defp build_prompt(input, past_messages) do
+    conversation_history =
+      Enum.reduce(
+        past_messages,
+        "",
+        fn %{sender: sender, message: message}, acc ->
+          "#{acc}#{sender}: #{message}\n"
+        end
+      )
```

Comment:
enter ---> [Enum.map_join/3](https://hexdocs.pm/elixir/Enum.html#map_join/3)! :)

Reply:
Done in https://github.com/Spectral-Finance/lux-spectra/pull/17/commits/d0e38c81e6b70c955ff9cc7eb1e171af339998c6 :)